packages/interactivity/src/hooks.tsx

```diff
@@ -1,12 +1,15 @@
+// @ts-nocheck
```

Comment:
We can add a task to finally migrate the whole runtime to TypeScript.

Reply:
I've seen you added the task to https://github.com/WordPress/gutenberg/pull/55459#issuecomment-1828761619. I'll mark this as resolved.